mike_skalandunas/gulpfile.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var gulp = require('gulp');
+var jshint = require('gulp-jshint');
+var mocha = require('gulp-mocha');
+var appFiles = ['index.js', 'lib/**/*.js', 'bin/**/*.js'];
+var testFiles = ['./test/**/*.js'];
+var watcher = gulp.watch('./**/*.js', ['jshint:test']);
+
+watcher.on('change', function(event) {
+  console.log('File ' + event.path + ' was ' + event.type + ', running tasks...');
```

Comment:
that's fine. I just havent seen that npm module. theres a gulp.watch already built into gulp